src/scripts/directives.js

```diff
@@ -325,19 +327,24 @@
                         scope.hourChoices = _.range(scope.hourMin, scope.hourMax, scope.hourStep);
                         scope.minuteChoices = _.range(scope.minuteMin, scope.minuteMax, scope.minuteStep);
                         var group = ['hours', 'minutes'];
+
+                        var cast = (scope.useDuration === 'true') ? moment.duration : moment;
```

Comment:
It seems wrong to check for the string `true`, could it be cast to a boolean `JSON.parse(scope.useDuration)`?